verify-service-provider.yml

```diff
@@ -29,6 +29,8 @@ serviceEntityIds: ${SERVICE_ENTITY_IDS:-[]}
 # To use the Verify Service Provider with multiple services, add their entity IDs here, e.g.
 # '["service-one-entity-id", "service-two-entity-id"]'
 
+# This applies if you're connecting using only the VSP
+# TODO Explain what this is and how it should be configured
```

Comment:
Best to ask a tech arch about this (John or Chris).  From my perspective, it's just an arbitrary string; I don't know what we recommend.

Reply:
This is for generating the PID. It uses a hashing algorithm in combination with the `hashingEntityId` to make sure the same user has the same PID (i.e. is always hashed the same way) and services can match them. As Alan mentioned above, it's only required if it's a multi-tenant set-up, otherwise it will use the serviceEntityId by default.